apps/os/sdk/cli/commands/gh-commands.ts

```diff
@@ -0,0 +1,16 @@
+import { z } from "zod";
+import { t } from "../config.ts";
+import { getRepoAccessToken } from "../git-utils.ts";
+
+export const ghPrintSetup = t.procedure
```

Comment:
Minor but I think slightly cleaner to do 

```ts
const ghPrintSetup = t.procedure...

export const githubRouter = t.router({
  setup: ghPrintSetup,
  someOtherCommandWeAddSomeday: otherCommand,
})
```

then keep the root router cleaner

```ts
const router = t.router({
  estate: { checkout: checkoutEstateCommand },
  gh: githubRouter,
});
```

Reply:
I probably should have done this with an `estateRouter`. easy to refactor anytime though so not a big deal